### ISSUE TITLE
No report for addEventListener on MediaQueryList on old browsers

### ISSUE THREAD

**Issue by evertbouw** [amilajack/eslint-plugin-compat#574]
I'm targeting Safari 13.1 and I'm not getting a warning for `addEventListener` on `MediaQueryList`

```js
window.matchMedia('prefers-color-scheme: dark').addEventListener("change", () => {});
```

It's only supported from Safari 14 and up: https://caniuse.com/mdn-api_mediaquerylist_eventtarget_inheritance
